apps/web/src/components/v2Editor/customBlocks/python/PythonOutput.tsx

```diff
@@ -80,6 +82,7 @@ export function PythonOutputs(props: Props) {
           key={i}
           className={clsx(
             ['plotly'].includes(output.type) ? 'flex-grow' : '',
+            !props.isDashboardView ? 'flex flex-col items-end' : '',
```

Comment:
Why is this needed? This makes all output align to the right, including text, it's looking weird now.
![image](https://github.com/user-attachments/assets/a9560639-46f8-4995-a408-27d0035f04c0)

Reply:
Oh, that's bad. It was made to move the PNG button (with relative position) to the right.

I can fix that right now in a new PR. Or if you want to, move these classes (with the ternary) to a new div inside the 'png' case in the PythonOutput component.

Reply:
@vieiralucas Fixed in #82.